.eslintrc

```diff
@@ -51,6 +52,243 @@
     "function-paren-newline": [0],
     "no-param-reassign": ["off"],
     "react/default-props-match-prop-types": ["off"],
-    "react/no-did-mount-set-state": ["off"]
-  }
+    "react/no-did-mount-set-state": ["off"],
+    "i18next/no-literal-string": ["error", {
+      "ignore": [
```

Comment:
Just an idea, but maybe we could use eslint's _extends_ to keep the ignore options in a separate file.

Reply:
Correct me if I'm wrong, but only shared modules can be used to extend eslint config, right?
Since those options are quite specific not sure if it's that beneficial to export it as a separate shared module.
And it would be faster for us to extend those whenever project changes and requires new exceptions.

Reply:
I didn't test it, but this: https://eslint.org/docs/user-guide/configuring#using-a-configuration-file reads as if it could be any file. I may be wrong here.

Reply:
Neat! It works 👍 
Pushing an update :)